src/common/reducers/userSummaries.js

```diff
@@ -56,6 +59,27 @@ export default function userSummaries(state = initialState, action) {
         isBusy: false,
       })
 
+    case REACTIVATE_USER_REQUEST:
+      return Object.assign({}, state, {
+        isBusy: true,
+      })
+    case REACTIVATE_USER_SUCCESS:
+      {
+        const userAttrs = action.response || {}
+        const userSummary = state.userSummaries[userAttrs.id] || {}
+        const user = Object.assign({}, userSummary.user, userAttrs)
+        const newUserSummary = Object.assign({}, userSummary, {user})
+        const userSummaries = Object.assign({}, state.userSummaries, {[user.id]: newUserSummary})
+        return Object.assign({}, state, {
+          isBusy: false,
+          userSummaries,
+        })
+      }
+    case REACTIVATE_USER_FAILURE:
+      return Object.assign({}, state, {
+        isBusy: false,
+      })
```

Comment:
Let's remove this logic for both the `REACTIVATE_USER_*` action types and the `DEACTIVATE_USER_*` types. In neither case is it intuitive for the user summary store state to be updated following success. We can get a refresh of the page by simply redirecting the user to `/users/:userId` after the graphQL API call is complete. See example at https://github.com/LearnersGuild/echo/blob/master/src/common/actions/project.js#L154.